### PR TITLE
Aurora serverless

### PR DESCRIPTION
**Pull request by pk0331**
Aurora serverless terraform module

### CODE REVIEW

**Comment by suryaoruganti** [comment]
All the same comments from #45 hold.

**Comment by suryaoruganti** [comment]
@PrashantRaj18198 plz review

**Comment by suryaoruganti** [comment]
the only difference between the serverless module and the provisioned module seems to be that one parameter - apart from potential value differences that are passed.

Merge this into the current `aurora` branch.

**Comment by prajjwaldimri** [comment]
@suryaoruganti @pk0331 Are we merging both of these modules? As that would impact how the code is structured in midgard.

**Comment by suryaoruganti** [comment]
as discussed - we are going to keep the two as different modules

**Comment by prajjwaldimri** [comment]
@pk0331 Please rename the folder to `aurora_serverless` as we are using underscore on other folders.

File: modules/aws/aurora-serverless/variables.tf [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+variable "copy_tags_to_snapshot" {
+  description = "Copy all Cluster `tags` to snapshots"
+  type        = bool
+  default     = true
+}
+
+variable "cluster_min_capacity" {
+  description = "Min cluster ACU"
+  type        = number
+}
```
[the remark this answers is not in the document]
Comment (pk0331) [reply modules/aws/aurora-serverless/variables.tf]: Done

File: modules/aws/aurora-serverless/variables.tf [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+variable "cluster_max_capacity" {
+  description = "Max cluster ACU"
+  type        = number
```
[the remark this answers is not in the document]
Comment (pk0331) [reply modules/aws/aurora-serverless/variables.tf]: Done

**Comment by suryaoruganti** [comment]
@pk0331 have you validated that it is working by connecting to the database and doing some operations?

**Comment by suryaoruganti** [comment]
> have you validated that it is working by connecting to the database and doing some operations?

@pk0331 @shubhamgrg04

**Comment by shubhamgrg04** [comment]
> 
Tested by connecting to aurora-mysql and aurora-postgres instance in preprod

**Comment by shubhamgrg04** [comment]
Merged https://github.com/argonautdev/tf-modules/pull/87

**Pull request closed without merging**